Source: https://bugs.kde.org/show_bug.cgi?id=290719
Upstream: yes
Reason: autoreconf fails with recent versions of automake
--- valgrind-3.7.0/Makefile.am
+++ valgrind-3.7.0/Makefile.am
@@ -61,7 +51,8 @@
 # default.supp, as it is built from the base .supp files at compile-time.
 dist_noinst_DATA = $(SUPP_FILES)
 
-pkglib_DATA = default.supp
+vglibdir = $(pkglibdir)
+vglib_DATA = default.supp
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = valgrind.pc
--- valgrind-3.7.0/coregrind/Makefile.am
+++ valgrind-3.7.0/coregrind/Makefile.am
@@ -495,7 +495,8 @@
 	m_gdbserver/powerpc-altivec64l.xml
 
 # so as to make sure these get copied into the install tree
-pkglib_DATA  = $(GDBSERVER_XML_FILES)
+vglibdir = $(pkglibdir)
+vglib_DATA  = $(GDBSERVER_XML_FILES)
 
 # so as to make sure these get copied into the tarball
 EXTRA_DIST  += $(GDBSERVER_XML_FILES)
